snake/common/variants/variants_snake.py

```diff
@@ -1,3 +1,70 @@
+# Freebayes calls a variant multiple times, when a bed file contains overlapping regions.
+# This rule uses bedtools merge to flatten a bedfile.
+
+rule flattenBedfileFreebayes:
```

Comment:
Could you rename that function since it is not only interesting for freebayes. Maybe something like: mergeRegionsInBed?